src/main/java/com/CreditPipeline/conveyor/controller/Conveyor.java

```diff
@@ -0,0 +1,90 @@
+package com.creditPipeline.conveyor.controller;
+
+import com.creditPipeline.conveyor.dto.CreditDTO;
+import com.creditPipeline.conveyor.dto.LoanApplicationRequestDTO;
+import com.creditPipeline.conveyor.dto.LoanOfferDTO;
+import com.creditPipeline.conveyor.dto.ScoringDataDTO;
+import com.creditPipeline.conveyor.exception.ScoringServiceException;
+import com.creditPipeline.conveyor.service.CalculateService;
+import com.creditPipeline.conveyor.service.OffersService;
+import com.creditPipeline.conveyor.service.ScoringService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+
+@RestController
+@RequestMapping("/conveyor")
+@Tag(name = "Конвеер")
+public class Conveyor {
+
+    private final ScoringService scoringService;
+    private final CalculateService calculateService;
+    private final OffersService offersService;
+    private final static Logger logger = LogManager.getLogger(Conveyor.class);
+
+    @Autowired
+    public Conveyor(ScoringService scoringService, CalculateService calculateService, OffersService offersService) {
+        this.scoringService = scoringService;
+        this.calculateService = calculateService;
+        this.offersService = offersService;
+    }
+
+    @PostMapping("/conveyor/calculation")
```

Comment:
Так как ты указал над классом @RequestMapping("/conveyor")
```suggestion
    @PostMapping("/calculation")
```